airgun/helpers/base.py

```diff
@@ -37,6 +37,4 @@ def form_fill_reader(self, navigation_name, navigation_kwargs=None, values=None,
             values = {}
         view = self.entity.navigate_to(self.entity, name=navigation_name, **navigation_kwargs)
         view.fill(values)
-        if not read_widget_names:
-            read_widget_names = list(values.keys())
         return view.read(widget_names=read_widget_names)
```

Comment:
so `return view.read(widget_names=None)` works fine?

Reply:
yes, if submitting read_widget_names will return only that widget values, else will read all the view widgets the same as regular read

Reply:
just checking =)